src/components/pages/devices/flyouts/deviceDetails/deviceDetails.scss

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+@import 'src/styles/mixins';
+@import 'src/styles/themes';
+
+.device-details-container {
+  @include rem-font-size(14px);
+  @include rem-fallback(line-height, 17px);
+  display: flex;
+  flex-flow: column nowrap;
+
+  .flyout-section {
+    .section-content {
+      @include rem-fallback(padding, 0px, 12px, 0px, 12px);
+
+      .basic-row {  @include rem-fallback(padding-top, 16px); }
+    }
+  }
+
+  .device-icon svg {
```

Comment:
Seems like this should be placed inside some kind of device details header section?

Reply:
Its possible that was there before I added the header section class.